internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/hadoop/TestVenicePushJob.java

```diff
@@ -635,6 +633,8 @@ public void testKIFRepushFetch(boolean chunkingEnabled) throws Exception {
             new UpdateStoreQueryParams().setStorageQuotaInByte(Store.UNLIMITED_STORAGE_QUOTA)
                 .setPartitionCount(2)
                 .setIncrementalPushEnabled(true)
+                .setWriteComputationEnabled(true)
```

Comment:
As discussed offline, either remove this or add a check to prevent non-chunking store to chunking store repush,